src/script/components/calling/GroupVideoGridTile.tsx

```diff
@@ -250,6 +251,23 @@ const GroupVideoGridTile: React.FC<GroupVideoGridTileProps> = ({
           {nameContainer}
         </div>
       )}
+      {doVideoReconnecting && (
+        <div className="group-video-grid__pause-overlay">
+          <div className="background">
+            <div className="background-image"></div>
+            <div className="background-darken"></div>
+          </div>
+
+          <div
+            className="group-video-grid__pause-overlay__label"
+            css={{fontsize: minimized ? '0.6875rem' : '0.875rem'}}
+            data-uie-name="status-video-paused"
+          >
+            {t('videoCallParticipantConnecting')}
+          </div>
+          {nameContainer}
+        </div>
+      )}
```

Comment:
this sounds a bit redundant code to me, maybe we could merge the common parts with what we already have

<img width="717" alt="image" src="https://github.com/user-attachments/assets/c6973f72-05ea-434e-8a93-ff80b3f7c61e" />

Reply:
Ok i will adjust the template